analysis/source/multivar_draw/multivar_draw.r

```diff
@@ -0,0 +1,22 @@
+library(yaml)
+CONFIG <- yaml.load_file("config_global.yaml")
+
```

Comment:
No need to rewrite this, but we have a typically format for organizing our R scripts. See [here](https://github.com/gslab-econ/ad-price-drivers/blob/master/analysis/source/analysis/estimation_adintel/plot_cb.R) for an example.

1. The first header loads all of the packages used in the script. 

2. The second headed loads the `config_global.yaml` and assign all of the paths we use to variable names. This way, if we change any paths, we only need to update this header rather than the body of the program.

3. The body of the code is then stored in a `main()` function which calls secondary functions that complete individual data manipulation or visualization tasks. The main function is then called at the end of the script. 

See the tidyverse  R code guide [here](https://style.tidyverse.org/index.html) for more minutiae.